test/fact_models.rb

```diff
@@ -23,6 +23,7 @@ class GameFactModel < ActiveReporting::FactModel
 end
 
 class SeriesFactModel < ActiveReporting::FactModel
+  dimension_hierarchy %i[name]
```

Comment:
I'd prefer coming up with another test case for this change that doesn't involve adding a hierarchical dimension to this fact model as this is forcing other unrelated tests to change.

Reply:
done, platform's hierarchy was not used in any test